pkgs/tools/text/txt2tags/default.nix

```diff
@@ -1,34 +1,20 @@
-{ lib, stdenv, fetchurl, python2 }:
+{ lib, buildPythonApplication, fetchPypi, tox }:
 
-stdenv.mkDerivation rec {
-  version = "2.6";
+buildPythonApplication rec {
   pname = "txt2tags";
+  version = "3.7";
 
-  dontBuild = true;
-
-  # Python script, needs the interpreter
-  propagatedBuildInputs = [ python2 ];
-
-  installPhase = ''
-    mkdir -p "$out/bin"
-    mkdir -p "$out/share/doc"
-    mkdir -p "$out/share/man/man1/"
-    sed '1s|/usr/bin/env python|${python2}/bin/python|' < txt2tags > "$out/bin/txt2tags"
-    chmod +x "$out/bin/txt2tags"
-    gzip - < doc/manpage.man > "$out/share/man/man1/txt2tags.1.gz"
-    cp doc/userguide.pdf "$out/share/doc"
```

Comment:
The github source (as opposed to the pypi) has some docs https://github.com/txt2tags/txt2tags/tree/3.7/docs though I suppose losing the docs isn't the end of the world. I just like not having to use the internet, and if possible not having to leave my terminal.

Either way up to you, looks good to me